Makefile

```diff
@@ -204,11 +204,7 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 ## Tool Versions
 arch:= $(shell uname -m)
 
-ifeq ($(arch), ppc64le)
-KUSTOMIZE_VERSION ?= v5.1.0
-else
-KUSTOMIZE_VERSION ?= v3.8.7
-endif
+KUSTOMIZE_VERSION ?= v5.2.1
```

Comment:
Just to confirm, does `v5.2.1` work for other architectures, like Macs?

Reply:
https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.2.1 supports Linux, Window$, and Mac

cc @rimolive